runbotics/runbotics-desktop/src/action/folder/folder.test.ts

```diff
@@ -75,6 +75,18 @@ describe('FolderActionHandler', () => {
         }
     };
 
+    const createTestFolder = () => {
+        if (!fs.existsSync(`${TEMP_FOLDER_PATH}${path.sep}${TEST_FOLDER_NAME}`)) {
+            fs.mkdirSync(`${TEMP_FOLDER_PATH}${path.sep}${TEST_FOLDER_NAME}`);
```

Comment:
expression ${TEMP_FOLDER_PATH}${path.sep}${TEST_FOLDER_NAME} is used 2 or even more times, make it a variable

Reply:
e.g. fullTestFolderPath